lib/fluent/event.rb

```diff
@@ -70,6 +88,10 @@ def repeatable?
       true
     end
 
+    def slice(index, num)
+      self.dup
```

Comment:
If `slice(1, 1)` is called, `slice` should return same object, not empty steram?